docs/application/web/api/5.5/device_api/mobile/tizen/notification.html

```diff
@@ -2344,7 +2344,6 @@ <h2 id="full-webidl">4. Full WebIDL</h2>
 <pre class="webidl prettyprint">module Notification {
   typedef DOMString NotificationId;
   enum NotificationType { "STATUS" };
-  enum StatusNotificationType { "SIMPLE", "THUMBNAIL", "ONGOING", "PROGRESS" };
```

Comment:
Just a question.

In https://github.com/Samsung/tizen-docs/pull/731#discussion_r291984362, it is said that this fix should be applied to 4.0 and 5.0 also.

I want to check if you are going to apply this fix to 4.0 and 5.0 version on master branch with another PR.

Reply:
yes, we are planning to add include this change to master branch as well, however in a separate change.